.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+	"extends": "eslint-config-jquery",
+	"root": true,
+	"rules": {
+
+		// Increase max-len to 120 for now, there are too many violations and the rule
+		// is not auto-fixable.
```

Comment:
It is valid. This type of ESLint config allows comments, similarly to how .jshintrc allowed comments.